src/System.Private.ServiceModel/tests/Common/Infrastructure/src/BridgeClientCertificateManager.cs

```diff
@@ -32,6 +34,20 @@ public static class BridgeClientCertificateManager
         private static Dictionary<string, CertificateCacheEntry> s_myCertificates = new Dictionary<string, CertificateCacheEntry>(StringComparer.OrdinalIgnoreCase);
         private static Dictionary<string, CertificateCacheEntry> s_rootCertificates = new Dictionary<string, CertificateCacheEntry>(StringComparer.OrdinalIgnoreCase);
 
+        private static readonly StoreLocation PlatformSpecificStoreLocation; 
+
+        static BridgeClientCertificateManager()
+        {
+            if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
+            {
```

Comment:
My API Catalog shows RuntimeInformation is not available on NET Native.  Have you tried building in TFS as NET Native yet?